openSearchLane.go

```diff
@@ -0,0 +1,231 @@
+package lane
+
+import (
+	"context"
+	"log"
+	"net/http"
+	"reflect"
+	"strings"
+	"sync"
+	"time"
+)
+
+const (
+	//Default threshold for logging messages
+	OslDefaultLogThreshold    = 100
+	//Default maximum buffer size for log messages 
```

Comment:
These comment are redundant with the name itself. It would be more helpful if they explained additional details or at least used alternate words.

`// Default number of log messages that are queued up before sending to OpenSearch in a bulk insert`